react/react-training/src/app/screens/Game/index.js

```diff
@@ -0,0 +1,83 @@
+import React, { Component } from 'react';
+
```

Comment:
Delete this break line

Reply:
Ignoro este error porque Es-Lint saca error. Que estos dos imports deben estar separados.